### ISSUE TITLE
First one in a list of arrow functions is in a different colour

### ISSUE THREAD

**Issue by laurent22** [babel/babel-sublime#325]
For example in here, `username_changeText` is white but `password_changeText` is green:

![](http://imgur.com/JXmGVZR.png)

and if I change the contructor into an arrow function, it's the one that becomes white:

![](http://imgur.com/0bdArze.png)

So it seems if there are several arrow functions one after another, the first one is rendered differently.

For information this is the code I'm using:

```
import React, { Component } from 'react';
import { View, Button, TextInput } from 'react-native';
import { connect } from 'react-redux'
import { Log } from 'src/log.js'

class LoginScreenComponent extends React.Component {
	
	static navigationOptions = {
		title: 'Login',
	};

	constructor = () => {
		super();
		this.state = { username: '', password: '' };
	}

	username_changeText = (text) => {
		this.setState({ username: text });
	}

	password_changeText = (text) => {
		this.setState({ password: text });
	}

	loginButton_press = () => {
		Log.info('LOGIN');
	}

}
```

**Comment by borela** [comment]
Try FJSX15 https://github.com/borela/naomi it has better support ES6 classes and stage-0 features.

**Comment by Thom1729** [comment]
This should be fixed in v10.0.0-beta.1. See #379 for more information on the new release. Please report any problems with v10 in a new issue.
